.storybook/preview.js

```diff
@@ -41,26 +42,39 @@ export const decorators = [
 
         const session = {
             user: {
-                id: "example_id",
-                name: "Name",
-                email: "example@mail.test",
+                id: 'example_id',
+                name: 'Name',
+                email: 'example@mail.test',
                 image: null,
-                role: "ADMIN",
-            }
-        }
+                role: 'ADMIN',
+            },
+        };
 
         const themeType = context.globals.theme;
+        const locale = context.globals.locale;
+
+        const messages = {
```

Comment:
This is not scalable approach. It would be better to have selector for local like for theme. Or we can use one locale for examples — `en`.

Reply:
Remove toggler